python/deptry/cli.py

```diff
@@ -224,6 +224,16 @@ def display_deptry_version(ctx: click.Context, _param: click.Parameter, value: b
     default={},
     show_default=False,
 )
+@click.option(
+    "--pep621-dev-dependency-groups",
+    "-ddg",
+    type=COMMA_SEPARATED_TUPLE,
+    help="""For projects that use PEP621 and that do not use a build tool that has its own method of declaring development dependencies,
+    the --pep621-dev-dependency-groups (-ddg) provides the option to specify which groups under [project.optional-dependencies] in pyproject.toml
```

Comment:
"the --pep621-dev-dependency-groups (-ddg) provides" - I guess this should have the word "option" before "provides"

Reply:
or, remove "the"